src/snowflake/cli/plugins/nativeapp/run_processor.py

```diff
@@ -242,6 +239,8 @@ def drop_application_before_upgrade(
                 )
                 raise typer.Exit(1)
         try:
+            cascade_msg = " (cascade)" if cascade else ""
+            cc.step(f"Dropping application object {self.app_name}{cascade_msg}.")
```

Comment:
Opportunistic fix; noticed there was no CLI output when an application is dropped during the upgrade flow.

Reply:
Weird that no snapshot needed to be updated. Gap in IT coverage?

Reply:
Yeah, I think by doing the x-upgrade test as a positive parameterized test we'll get a ton more coverage here. But also I think that integration tests don't do snapshot testing unless you ask? I'll check